schema/PACK.xsd

```diff
@@ -657,7 +657,7 @@
     <xs:attribute name="title" type="xs:string" use="required" />
     <!-- if true, the vendor gives permission for this file being extracted from the pack and displayed on a web-page -->
     <!-- links to web pages are assumed public -->
-    <xs:attribute name="public" type="xs:boolean" use="optional" default="false" />
+    <xs:attribute name="public" type="xs:boolean" use="optional" default="true" />
```

Comment:
I think we should update the schema version, don't we?

Reply:
Yes, didn't think about that.